docs/migration-guide.md

```diff
@@ -6,7 +6,29 @@ If you are starting from scratch, see the [new project guide](setting-up-a-gucdk
 
 ---
 
-## Account readiness
+## Overall migration process
+
+1. Identify the [GuCDK pattern](https://guardian.github.io/cdk/modules/patterns.html) which works for your app
+1. Check account readiness
+1. Add GuCDK to your repository and CI/CD process
+1. Instantiate a GuCDK pattern alongside your legacy infrastructure (dual-stack), for stateless resources
+1. Switch your production workload (e.g. serving HTTP traffic or processing data) to the GuCDK infrastructure
+1. Remove legacy infrastructure for stateless resources
+1. Where necessary, define stateful resources (e.g. DBs) via cdk components instead of via the original template
+
+## Identify the right GuCDK pattern
+
+| YAML Infrastructure                                                                   | Application type                    | GuCDK pattern to use            |
+|---------------------------------------------------------------------------------------|-------------------------------------|---------------------------------|
+| `AWS::AutoScaling::AutoScalingGroup` with `AWS::ElasticLoadBalancing::LoadBalancer`   | Serving HTTP traffic                | `GuEc2App`                      |
```

Comment:
Did you experiment with the readability of merging these two? I had to do a double take between the rows to clock that the difference is just the version, but maybe the merged version is a bit too busy?
```suggestion
| `AWS::AutoScaling::AutoScalingGroup` with either `AWS::ElasticLoadBalancing::LoadBalancer` or `AWS::ElasticLoadBalancingV2::LoadBalancer`   | Serving HTTP traffic                | `GuEc2App`                      |
```

Reply:
I hadn't experimented with this before but I think the merged version is better so I've updated this. Thanks for the suggestion!